lib/carrier/credentials.ex

```diff
@@ -1,19 +1,14 @@
 defmodule Carrier.Credentials do
 
-  defstruct [:id, :private, :public, :tag]
+  defstruct [:id, :tag]
 
   @cred_db "carrier_credentials.db"
-  @private_key "carrier_priv.key"
-  @public_key "carrier_pub.key"
   @carrier_id "carrier.id"
 
-  # 32 byte key w/64 byte checksum
-  @key_hash_size 64
-
   @doc "Generates a new private/public keypair"
```

Comment:
Is this even needed anymore, then? At the very least, the docs (and potentially the name) for the function should change.